src/seq/fasta.rl

```diff
@@ -0,0 +1,251 @@
+
+
+# FASTA sequence types
+
+@doc """
+Metadata for FASTA sequence records containing just a `description` field.
+""" ->
+type FASTAMetadata
+    description::String
```

Comment:
It's a reasonable suggestion. In the BED parser I ended up making most of the fields Nullable, since most of the columns in BED are optional. Making the description Nullable here might be the most consistent thing to do.